internal/project/common/all.go

```diff
@@ -28,6 +29,10 @@ func NewAll(meta *meta.Options) *All {
 
 // CompileMakefile implements makefile.Compiler.
 func (all *All) CompileMakefile(output *makefile.Output) error {
+	if all.meta.ContainerImageFrontend != config.ContainerImageFrontendDockerfile {
```

Comment:
this _should_ still work same way for `Pkgfile` builds, given that the tree is built correctly

Reply:
it was adding `all: pkgfile build`

not sure where it came from (`pkgfile` was the dag name)

Reply:
scratch that